tests/rules/test_enabled.py

```diff
@@ -101,16 +101,8 @@ async def test_call(self):
                 assert len(model.rules) == 1
                 rule_name = f"Enabled.{name}:{index}"
                 rule = model.rules[rule_name]
+                rule.alarm.severity_queue = asyncio.Queue()
```

Comment:
Why not call `init_severity_queue` here?

Reply:
Good suggestion.